src/test/java/uk/ac/cam/cl/dtg/segue/api/managers/GroupManagerTest.java

```diff
@@ -110,11 +122,11 @@ public final void groupManager_createValidGroup_aGroupShouldBeCreated() {
       replay(this.userManager, this.groupDataManager, this.dummyMapper);
 
       // check that the result of the method is whatever comes out of the database
-      UserGroupDTO createUserGroup = gm.createUserGroup(someGroupName, someGroupOwner);
+      UserGroupDTO createUserGroup = this.groupManager.createUserGroup(someGroupName, someGroupOwner);
 
       // check that what goes into the database is what we passed it.
-      assertEquals(capturedGroup.getValue().getOwnerId(), someGroupOwner.getId());
-      assertEquals(capturedGroup.getValue().getGroupName(), someGroupName);
+      assertEquals(someGroupOwner.getId(), capturedGroup.getValue().getOwnerId());
+      assertEquals(someGroupName, capturedGroup.getValue().getGroupName());
       assertTrue(capturedGroup.getValue().getCreated() instanceof Date);
```

Comment:
Since you've fixed the two assertions above, you can also change this one to `assertInstanceOf`.

Reply:
Oooh, there's a dedicated assertion for that?